### ISSUE TITLE
According to the example for semantic segmentation, there is no problem in the training set, but the size of the validation set is always wrong 

### ISSUE THREAD

**Issue by Dingkun1219** [qubvel/segmentation_models.pytorch#516]
When I was training my own data set, according to the sample you provided, it can be implemented on the training set, but the validation set always reports an error (dimension error). I checked the code and found that it was because of the validEpoch class in utils.train problem. For example, a picture of (1, 3, 448, 640) is output by self.model.forword() of batch_updata in validEpoch as (1, 1, 447, 639), that is, the size is 1 smaller than the original image .
Can you help me solve this problem?
